### DISCUSSION TITLE
[Feature] Ability to add tags to media from shared album.

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#14238] [Feature Request]
Think of the situation user A and user B are sharing photos using a shared album. At the moment user A can only tag his own photos in the shared album. It would be a nice feature that user A can tag his own photos and the photos shared by user B.

**Comment by danieldietzler** [comment]
Duplicate of #12614

**Reply by ghost** [reply]
Thanks @danieldietzler
